### ISSUE TITLE
Portals only work one way

### ISSUE THREAD

**Issue by tristanpoland** [Multiverse/Multiverse-NetherPortals#229]
I recently downloaded this plugin on my Apex Hosting server, and followed the steps to link  each of my several worlds to its own nether and end dimension, after setting this up I entered a portal, and it worked fine, but when I tried to go back, it told me that the nether did not have a linked nether dimension.

**Comment by benwoo1110** [comment]
Did you link them both ways? So example you need to do 2 command /mvp link overworld nether and /mvp link nether overworld

**Comment by LazyGon** [comment]
Even if worlds are linked properly, you have to allow the default level nether to link overworld -> nether.

**Comment by benwoo1110** [comment]
Yes that the behaviour of the server software, there is nothing mv can do.

**Comment by benwoo1110** [comment]
This has been fixed with Paper by using EntityPortalReadyEvent
